src/Symfony/Component/DependencyInjection/Argument/RewindableGenerator.php

```diff
@@ -14,13 +14,21 @@
 /**
  * @internal
  */
-class RewindableGenerator implements \IteratorAggregate
+class RewindableGenerator implements \IteratorAggregate, \Countable
 {
     private $generator;
+    private $count;
 
-    public function __construct(callable $generator)
+    /**
+     * RewindableGenerator constructor.
+     *
+     * @param callable $generator
+     * @param int|null $count     If provided, used as the returned value for count. iterator_to_array will be used otherwise.
+     */
+    public function __construct(callable $generator, $count = null)
```

Comment:
Yes, it confirms https://github.com/symfony/symfony/pull/21455#discussion_r98360789